kinetic/distribution.yaml

```diff
@@ -1933,6 +1933,12 @@ repositories:
       url: https://github.com/piraka9011/dialogflow_ros.git
       version: kinetic-devel
     status: developed
+  dmu_ros:
+	doc:
+		type: git
+		url: https://github.com/leo-muhendislik/dmu_ros.git
+		version: kinetic-devel
+	status: developed
```

Comment:
You need to use spaces for the indentation as for the rest of the file in order to pass the CI tests.